app/web/service/notifications.ts

```diff
@@ -65,3 +69,32 @@ export async function registerPushNotificationSubscription(
 export async function sendTestPushNotification() {
   await client.notifications.sendTestPushNotification(new Empty());
 }
+
+export async function listNotifications() {
+  const req = new ListNotificationsReq();
+  const res = await client.notifications.listNotifications(req);
+  return res.toObject();
+}
+
+export async function markAllNotificationsSeen(
+  lastestNotificationId: Notification.AsObject["notificationId"],
```

Comment:
Should this just be an integer, no?

Reply:
Yep, it's just saying use the same type as `Notification.AsObject`, which is a number:

<img width="270" alt="Screenshot 2025-04-23 at 7 17 31 PM" src="https://github.com/user-attachments/assets/a3cf63cc-b33d-4e8c-922b-b2e579a0d8a2" />